cluster/saltbase/salt/docker/docker.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Docker Application Container Engine
+Documentation=http://docs.docker.com
+After=network.target docker.socket
+Requires=docker.socket
+
+[Service]
+EnvironmentFile={{ environment_file }}
+ExecStart=/usr/bin/docker -d -H fd:// "$DOCKER_OPTS"
+MountFlags=slave
+LimitNOFILE=1048576
+LimitNPROC=1048576
+LimitCORE=infinity
+Restart=always
+RestartSec=2s
+StartLimitInterval=0
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
@justinsb - or better, in case someone does a yum update, can you only write this file if the os_family is not RedHat since the RedHat packaged docker does include a service file?  so basically, this is the only os specific issue among systemd distros right now.